deploy-lighthouse-reth.py

```
@@ -0,0 +1,866 @@
+            print(f">> Successfully downloaded: {asset['name']}")
+
+        except requests.exceptions.RequestException as e:
+            print(f"Error: Unable to download file. Try again later. {e}")
+            exit(1)
+
+        # Extract the binary to the home folder
+        with tarfile.open(f"{tar_filename}", 'r:gz') as tar:
+            tar.extractall()
+
+        # Move the binary to /usr/local/bin using sudo
+        os.system(f"sudo mv reth /usr/local/bin")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)